docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -1604,6 +1604,57 @@ public class OidcStartup {
 For more complex setup involving multiple tenants please see the xref:security-openid-connect-multitenancy.adoc#programmatic-startup[Programmatic OIDC start-up for multitenant application]
 section of the OpenID Connect Multi-Tenancy guide.
 
+== OAuth 2.0 Step Up Authentication
```

Comment:
```suggestion
== Step Up Authentication
```

Just a minor suggestion as we also refer to DPoP and other OAuth2 specs without `OAuth2` in the section title, given the doc is about OIDC :-)